### PR TITLE
clean up console logging

### PR DESCRIPTION
**Pull request by balmas**
for alpheios-project/components#16

See PR alpheios-project/components#740 for discussion

### CODE REVIEW

File: src/grammars.js [hunk 1]
```diff
@@ -57,7 +55,6 @@ export default class Grammars {
 
       return requests
     } catch (error) {
-      console.log(`Unable to fetch resources due to: ${error}`)
```
Comment (irina060981) [inline src/grammars.js:60]: May be it is worth to save this error to console?

**Review by irina060981 — APPROVED** [review]
Looks ok

**Review by kirlat — COMMENTED** [review]
The same question here: do we want to use a common Alpheios prefix?

**Review by kirlat — APPROVED** [review]

File: src/grammars.js (hunk 1 above)
Comment (balmas) [reply src/grammars.js:60]: ok, will put it back.

**Pull request merged**